app/helpers/frontend_helper.rb

```diff
@@ -1,2 +1,13 @@
 module FrontendHelper
+  def scholarship_style_change(scholarship)
```

Comment:
that i made a comment on this before actually i have no clue to write helpers  I waned to discuss of those